Backpack-SwiftUI/CarouselCard/Classes/BPKCarouselCard.swift

```diff
@@ -0,0 +1,86 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2024 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import SwiftUI
+
+public struct BPKCarouselCard<Content: View>: View {
+    @ViewBuilder let content: () -> Content
+    let title: String
+    let description: String
+    let imageAccessibilityLabel: String
+    
+    public init(
+        content: @escaping () -> Content,
+        title: String,
+        description: String,
+        imageAccessibilityLabel: String
+    ) {
+        self.content = content
+        self.title = title
+        self.description = description
+        self.imageAccessibilityLabel = imageAccessibilityLabel
+    }
+    
+    public var body: some View {
+        GeometryReader { reader in
+            BPKCard(padding: .none, cornerStyle: .large, elevation: .focus, content: {
+                VStack(alignment: .leading) {
+                    content()
+                        .frame(
+                            width: reader.size.width,
+                            height: reader.size.height * 0.60
+                        )
+                        .clipped()
+                    
+                    VStack(alignment: .leading) {
```

Comment:
I think we will need spacing: .zero to remove the OS default spacing between the two labels
https://acodeguy.medium.com/the-hidden-gotcha-of-default-vstack-spacing-c3d7dc01466

```suggestion
                    VStack(alignment: .leading, spacing: 0) {
```

Reply:
should be 
```suggestion
                    VStack(alignment: .leading, spacing: BPKSpacing.none) {
```